src/main/java/com/wiley/driver/factory/capabilities/ChromeCaps.java

```diff
@@ -28,6 +30,11 @@ public ChromeOptions get() {
         ChromeOptions caps = getChromeOptions();
         if (!this.customCaps.asMap().isEmpty()) {
             caps.merge(this.customCaps);
+
+            // This is a workaround for ChromeOptions.merge() issue when passed arguments are not being assigned properly
+            Map chromeCapability = (Map) caps.getCapability(ChromeOptions.CAPABILITY);
+            List argumentsList = (List) chromeCapability.get("args");
```

Comment:
we don't add types references to the variable name. rename argumentsList to arguments